### ISSUE TITLE
multiple menu doesn't work properly after Ionic 2.2.0 upgrade

### ISSUE THREAD

**Issue by kahveci** [driftyco/ionic#10986]
**Ionic version:**  (check one with "x")
[ ] **1.x**
[ X] **2.x**

**I'm submitting a ...**  (check one with "x")
[ X] bug report
[ ] feature request
[ ] support request => Please do not submit support requests here, use one of these channels: https://forum.ionicframework.com/ or http://ionicworldwide.herokuapp.com/

**Current behavior:**
The current behaviour has started after upgrading my Ionic app from 2.0.1 to 2.2.0. So, this issue can be considered as a breaking change. 

In my app, I have 2 tabs which have their own menus. I have applied the same pattern suggested in GitHub. Shortly, when I move from tab1 to tab2 and press the menu icon in tab2, there is no response in tab2! If you press the same icon second time, it works properly and the menu appears in tab2. After that point, it keeps working properly in tab2. However, if you move to tab1, the same strange behaviour is applied to the tab1. When you click twice, it starts working properly again. Of course, the loop goes on if you move to tab2 and press to the menu icon again.

I checked the tab1 when I first pressed the menu icon in tab2. I realised that the first button press always opens the menu in the previous tab. Moreover, this happens despite the menu is disabled in that particular tab. For example: 

```
  // function to open the menu in tab2
  openMenu() {
    this.menu.enable(true, 'tab2-menu'); // opens the menu in tab1 for the first click
    console.log("tab1-menu - "+this.menu.isEnabled('tab1-menu')); // false
    console.log("tab2-menu - "+this.menu.isEnabled('tab2-menu')); // true

    this.menu.open();
  }
```

**Expected behavior:**
When clicking on a menu element, only the associated menu should take place.

**Steps to reproduce:**
Create two tabs and one menu for each. Apply the multiple menu pattern as shown below:
https://github.com/driftyco/ionic/tree/master/src/components/menu/test/multiple

**Comment by jgw96** [comment]
Hello, thank for opening an issue with us, we will look into this.

**Comment by jgw96** [comment]
Hello, thanks for opening an issue with us! This has been fixed and the fix is in the latest release (3.0.1).
